src/windows-nano/1803/helix/amd64/Dockerfile

```diff
@@ -0,0 +1,33 @@
+FROM mcr.microsoft.com/powershell:6.2.0-nanoserver-1803
+
+ENTRYPOINT C:\Windows\System32\cmd.exe
+
+WORKDIR C:\\Setup
+
+# Get remote files
+ADD "https://www.nuget.org/api/v2/package/python/3.7.3" python-3.7.3-nuget-amd64.zip
+
+USER ContainerAdministrator 
+RUN setx /M PATH "%PATH%;C:\Program Files\PowerShell;C:\Python;C:\python\scripts" \
```

Comment:
From a logical perspective, it feels like adding these environment variables belong after python is actually installed.

Reply:
done